### PR TITLE
Filipino translations

### PR DESCRIPTION
**Pull request by IverCoder**
Filipino translations for both app and stub.

### CODE REVIEW

**Comment by yujincheng08** [comment]
```
/home/runner/work/Magisk/Magisk/app/src/main/res/values-tl/strings.xml:199:4: Failed to flatten XML for resource 'unsupport_magisk_msg' with error: Invalid unicode escape sequence in string
```

**Comment by IverCoder** [comment]
So how can we fix this lol

File: app/src/main/res/values-tl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    <string name="patch_file_msg">Pumili ng raw image (*.img) o ODIN tarfile (*.tar)</string>
+    <string name="reboot_delay_toast">Magre-reboot sa 5 segundo…</string>
+
+    <!--Superuser-->
+    <string name="su_request_title">Hiling para sa Superuser</string>
+    <string name="touch_filtered_warning">Hindi maberipika ng Magisk ang hiling para sa Superuser dahil may app na humaharang dito</string>
+    <string name="deny">Tanggihan</string>
+    <string name="prompt">Tanungin</string>
+    <string name="grant">Payagan</string>
+    <string name="su_warning">Nagbibigay ng full access sa iyong device.\Tanggihan kung hindi ka sigurado!</string>
```
Comment (usrDottik) [inline app/src/main/res/values-tl/strings.xml:50]: `\T`?

Comment (Stillhard) [reply app/src/main/res/values-tl/strings.xml:50]: Nice catch 👍🏼

Comment (IverCoder) [reply app/src/main/res/values-tl/strings.xml:50]: How can we fix this?

**Comment by IverCoder** [comment]
Bruh can somebody fix this for me 💀

**Comment by Stillhard** [comment]
> \Tanggihan

Try to remove the \ from that word as @ghost's suggestion at line 50

**Pull request closed without merging**